src/ansys/pytwin/evaluate/tbrom.py

```diff
@@ -39,8 +48,10 @@ def __init__(self, tbrom_name: str, tbrom_path: str):
         self._infmcs = None
         self._outmcs = None
         self._infbasis = None
-        self._outbasis = None
         self._pointsdata = None
+        self._meshdata = None
```

Comment:
We have 4 heavy memory internal variables for each tbrom. Should-we provide a way to clear a bit of memory from the TwinModel API at user request? Typically adding a method like `TwinModel.clear_tbrom_memory(tbrom_name, pointsdata:bool, meshdata:bool, modes:bool)` and later in a workflow, enabling to recover memory from available files (for the modes and points data typically) after a call to the `clear_tbrom_memory`. That would be a smart way to optimize memory management at user request. @chrpetre @EDCarman What do you think?

Reply:
as discussed : we might want to consider such approach in the future once we have more feedbacks on current workflows and possible limitations